apollo-cache-interceptor/gradle.properties

```diff
@@ -0,0 +1,3 @@
+POM_ARTIFACT_ID=apollo-cache-interceptor
```

Comment:
why not just call it `apollo-cache`? I thought that everything that belongs to normalize cache will be packaged in one artifact?

Reply:
Excellent question! I'm not 100% sure where I'm going with this but it feels like we should bundle the sqlite cache by default so multiple artifacts would be needed. At this stage of development, my rationale was that having multiple single-responsibility modules makes for better separation of concern and that we would "polish" the user-facing API/artifacts once everything is feature complete.

Reply:
Sure, we can always change this later, let's leave it for now.